### PR TITLE
GCP configuration for foxpass-ipsec-vpn

### PR DESCRIPTION
**Pull request by nshttpd**
Packer configuration file and moving things around that maybe specific for installs from cloud providers and/or images. Also updated config.py to check to see if we're running on an AWS instance or GCP instance to query the appropriate endpoints on the metadata server for information.

### CODE REVIEW

File: scripts/config.py [hunk 1]
```diff
@@ -98,8 +110,15 @@ def gather_data():
...
-    private_ip = urlopen('http://169.254.169.254/latest/meta-data/local-ipv4').read()
+    if is_gce():
+        headers = {'Metadata-Flavor': 'Google'}
+        request = Request(METADATA_BASE_URL + 'computeMetadata/v1/instance/network-interfaces/0/access-configs/0/external-ip', headers=headers)
+        public_ip = urlopen(request).read()
+        request = Request(METADATA_BASE_URL + 'computeMetadata/v1/instance/network-interfaces/0/ip', headers=headers)
+        private_ip = urlopen(request).read()
+    else:
+        public_ip = urlopen('http://169.254.169.254/latest/meta-data/public-ipv4').read()
+        private_ip = urlopen('http://169.254.169.254/latest/meta-data/local-ipv4').read()
```
Comment (aren) [inline scripts/config.py]: Any reason not to make these two lines use METADATA_BASE_URL also?

Comment (nshttpd) [inline scripts/config.py]: No, not really. I was trying to not touch as much of the "original" stuff as possible. I can do back and update that though too if you want.

**Comment by nshttpd** [comment]
@aren updated that in the config.py

**Comment by aren** [comment]
Thanks Steve!

**Pull request merged**